### PR TITLE
PedersenHash implementation

### PR DESCRIPTION
**Pull request by ch9xy**
I did not know where to put it so I put it there lol sorry on that :D

### CODE REVIEW

File: ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/EmptyNode.java [hunk 1]
```diff
@@ -32,3 +35,72 @@ public void insert(Bytes32 key, Bytes32 value) throws Exception {
         throw new UnsupportedOperationException("Unimplemented method 'insert'");
     }
 }
+
+
+
+public class PedersenHash {
+    private static final String HASH_ALGORITHM = "SHA-256";
```
Comment (dragan2234) [inline ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/EmptyNode.java:42]: @ch9xy I think we don't need a sha256 here, but maybe I'm wrong? Can you give a bit more explanation?

Comment (agnxsh) [reply ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/EmptyNode.java:42]: We don't need SHA256 here

**Comment by ch9xy** [comment]
I came up with a new code. I will be creating a new request.

File: ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/EmptyNode.java (hunk 1 above)
Comment (ch9xy) [reply ipa-multipoint/src/main/java/org/hyperledger/besu/ethereum/trie/verkle/EmptyNode.java:42]: > @ch9xy I think we don't need a sha256 here, but maybe I'm wrong? Can you give a bit more explanation?

it was just some placeholder algorithm for computation of commitment root

**Pull request closed without merging**